Vagrantfile

```diff
@@ -0,0 +1,34 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64" 
+  config.vm.provision "shell", inline: <<-SHELL
+    sudo apt-get update
+    sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev cifs-utils python-openssl git
+    
+    git config --global core.compression 9
```

Comment:
When trying to improve the performance of git clone in a situation like this or a build pipeline, an easy win is `git clone --depth 1 ...`